components/carousel/module/CarouselInfo.vue

```diff
@@ -84,5 +84,5 @@ const showPrice = computed((): boolean => {
   return Number(price.value) > 0 && !isCollection
 })
 
-const unitSymbol = computed(() => prefixToToken[props.item.chain || 'ksm'])
+const unitSymbol = computed(() => prefixToToken[props.item.chain || 'ahp'])
```

Comment:
```suggestion
const unitSymbol = computed(() => prefixToToken[props.item.chain || 'dot'])
```

Reply:
I can't see the deficiencies here as they all return `DOT`
<img width="256" alt="image" src="https://github.com/user-attachments/assets/9f567ec2-1c0a-483a-bff0-6f37ef63a87e">